docs/ipfs_datatypes.md

```diff
@@ -63,15 +63,19 @@ Objects may be explored with the IPLD explorer: https://explore.ipld.io/
 
 Within IPLD, the following data formats are currently usage within IPFS:
 
-1. `dag-pb`: UnixFS protobuf format for files, raw data, HAMT and directories.
+1. `dag-pb`: Protobuf wrapper around UnixFS protobuf format for files, raw data, HAMT, and directories.
```

Comment:
Protobuf wrapper around protobuf format? Double protobuf?

Reply:
Yeah, the `Data` field of the [dag-pb protobuf](https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-pb.md) holds the UnixFS protobuf. And it seems like this is always the case. I was quite puzzled too, which is why I went through the hassle of manually decoding the blocks and trying to dig up the dag-pb spec 🙃 

Sounds a bit weird though, I'm open to suggestions making it sound nicer!